codesy/views.py

```diff
@@ -1,11 +1,25 @@
 from django.views.generic import TemplateView
+from django.conf import settings
+
+
+def add_cc_debug(ctx):
+    if settings.DEBUG:
+        cc_debug = {
+            'cc_number': '4111111111111111',
+            'cc_ex_month': '01',
+            'cc_ex_year': '2020',
+            'cvc': '123',
+        }
+        ctx['cc_debug'] = cc_debug
+    return ctx
 
 
 class Home(TemplateView):
     template_name = 'home.html'
 
     def get_context_data(self, **kwargs):
         ctx = super(Home, self).get_context_data(**kwargs)
+        ctx = add_cc_debug(ctx)
```

Comment:
Can we make this:
```
ctx['cc_debug'] = cc_debug()
```
So we're explicit about what key is going into the context in `get_context_data`?